### ISSUE TITLE
Unable to build for web

### ISSUE THREAD

**Issue by marwfair** [pikaju/flutter-maps-launcher#6]
With version 1.2.2+1, I'm unable to build the app for web. I am getting this error:

`import 'package:maps_launcher/maps_launcher_web.dart';                                                             
       ^                                                                                                           
lib/generated_plugin_registrant.dart:22:54: Error: Getter not found: 'MapsLauncherWeb'.                            
  MapsLauncherWeb.registerWith(registry.registrarFor(MapsLauncherWeb)); `

1.2.2 worked fine for web, just not for iOS.

**Comment by Poujhit** [comment]
I too have the same problem.

**Comment by jrmarkham** [comment]
Same problem too

**Comment by vadrian89** [comment]
Hello, I have the same issue.

```
Doctor summary (to see all details, run flutter doctor -v):
[✓] Flutter (Channel beta, 1.21.0-9.2.pre, on Mac OS X 10.15.6 19G2021, locale
    ro-RO)
 
[✓] Android toolchain - develop for Android devices (Android SDK version 29.0.3)
[✓] Xcode - develop for iOS and macOS (Xcode 11.7)
[✓] Chrome - develop for the web
[✓] Android Studio (version 4.0)
[✓] VS Code (version 1.49.0)
[✓] Connected device (3 available
```

**Comment by RobProductions** [comment]
Same issue is happening to me, the plugin registrant is showing an error for a web class saying it doesn't exist. Builds fine on iOS and Android.

**Comment by DanielCardonaRojas** [comment]
Same here, look like this isn't conforming to Platform Interface: 

https://medium.com/flutter/how-to-write-a-flutter-web-plugin-part-2-afdddb69ece6

- Add the <plugin name>_platform_interface package.
- Migrate the plugin to use the platform interface.
- Add a <plugin name>_web package that extends the platform interface.

**Comment by basel-sader** [comment]
Facing same problem, can't build for web.

Anybody found a solution?

**Comment by MattyBoy4444** [comment]
The PR #8 has the fix.  I am just set my pubspec.yaml to use the PR git repository.

**Comment by pikaju** [comment]
I merged #8 and published a new version: `1.2.2+2`. Sorry for the wait, everyone.
